### PR TITLE
first draft of the themes documentation

### PR DESCRIPTION
**Pull request by davidplumb**

### CODE REVIEW

File: tables/themes.md [hunk 1]
```diff
@@ -1 +1,28 @@
-placeholder
+
+# tables/themes
+
+Note: Currently the `themes` table is a WIP so this information will need to be updated
+
+The `themes` table exists to help categorise topics into different parent themes that are represented with an id. This will help end users find common topics that they are interested in querying. It is still a WIP but eventually there will be something like `theme_id` in the `topics` table that will relate the two in a one to many relationship.
+
```
Comment (tomgiddingsjisc) [inline tables/themes.md:9]: Would be good to have an example here similar to the one in top_level_geographies? (under a header of 'Example use') Sometimes aids understanding for people when it's placed in context.

File: tables/themes.md [hunk 2]
```diff
@@ -1 +1,28 @@
-placeholder
+
+# tables/themes
+
+Note: Currently the `themes` table is a WIP so this information will need to be updated
+
+The `themes` table exists to help categorise topics into different parent themes that are represented with an id. This will help end users find common topics that they are interested in querying. It is still a WIP but eventually there will be something like `theme_id` in the `topics` table that will relate the two in a one to many relationship.
```
Comment (tomgiddingsjisc) [inline tables/themes.md:8]: What do you think to standardising the headers themselves? So having 'What are themes?' and structuring this way? (reasoning being to allow people to quickly jump to the information they need)

**Review by tomgiddingsjisc — COMMENTED** [review]
A couple of suggestions on changes with some reasoning - shout if you'd like to discuss anything there :)

File: tables/themes.md (hunk 2 above)
Comment (davidplumb) [reply tables/themes.md:8]: sure can standardise it like that, I think that would be a good idea

File: tables/themes.md (hunk 1 above)
Comment (davidplumb) [reply tables/themes.md:9]: think the problem atm is that it needs a foreign key setup for it to be useful at this point? as without a reference in topics it's hard to do an example involving other tables

Comment (tomgiddingsjisc) [reply tables/themes.md:9]: Ah that's fair - maybe worth putting the header in there with TBC or similar? Then we won't forget down the line!

Comment (davidplumb) [reply tables/themes.md:9]: yea good idea, I'll add it in then

File: tables/themes.md (hunk 2 above)
Comment (davidplumb) [reply tables/themes.md:8]: done

File: tables/themes.md (hunk 1 above)
Comment (davidplumb) [reply tables/themes.md:9]: done

**Review by tomgiddingsjisc — APPROVED** [review]
Excellent!

**Pull request merged**